example_allocators/compartment_alloc/README.md

```diff
@@ -0,0 +1,17 @@
+# Compartmentalising Allocator
+
+A modification of the "bump allocator" in this repo.
```

Comment:
Minor nitpicking. Normally we format text files to 80 chars.

Reply:
Fixed in [9c09bab](https://github.com/capablevms/cheri-examples/pull/88/commits/9c09babd299c9b6996d49a2ecfc63647a12e528c)!